packages/fetchai/skills/generic_seller/behaviours.py

```diff
@@ -148,8 +148,7 @@ def register_service_personality_classification(self) -> None:
         strategy = cast(GenericStrategy, self.context.strategy)
         descriptions = [
             strategy.get_register_service_description(),
-            strategy.get_register_personality_description(),
-            strategy.get_register_classification_description(),
+            strategy.get_registration_description(),
         ]
         oef_search_dialogues = cast(
             OefSearchDialogues, self.context.oef_search_dialogues
```

Comment:
This loop below is a problem btw. It does not play well with the logic you submitted earlier this week. If both fail, then the first failed message can be overwritten with the second...

Reply:
ok, the second bit can be broken down and moved into the SUCCESS and ERROR handlers

Reply:
Will have to do that in all skills using the same pattern